wp-content/themes/cpipr/css/style.css

```diff
@@ -1535,6 +1537,9 @@ blockquote p {
   .wrapper-special .container-special > .col-xs-6 {
     width: 100%;
   }
+  .wrapper-post-serie {
```

Comment:
@benlk should this be `.wrapper-post-series` rather than `.wrapper-post-serie`?

Reply:
https://github.com/INN/umbrella-cpipr/pull/27/files#diff-491875b64a837adea64560423c9819bcR155 has the class name as `wrapper-post-serie`, so this is correct.